example/index.html

```diff
@@ -38,6 +24,7 @@
   //provide listeners for notifications.
   $fh.sync.notify(function(notification){
     var code = notification.code
+    console.log("Sync ", notification);
```

Comment:
That's example application. Code is from documentation. We going to move this out of this repo and improve in different ticket.